### PR TITLE
New claim submissions with image uploading

### PR DESCRIPTION
**Pull request by mullerfourie**

### CODE REVIEW

File: src/components/form/DynamicForm.tsx [hunk 1]
```diff
@@ -48,18 +43,19 @@ export interface Callbacks {
...
+		imageList: [],
 	};
 
 	async componentWillMount() {
 		let hiddenCount = 0;
-		console.log('Dynamic From: ' + this.props.formSchema);
+		// console.log('Dynamic From: ' + this.props.formSchema);
 		this.props.formSchema.map((field: any) => {
-			console.log('Field: ' + field.label);
+			// console.log('Field: ' + field.label);
```
Comment (Nicolaas0411) [inline src/components/form/DynamicForm.tsx]: Can we please remove the commented out code. (Makes the code look a bit cluttered)

File: src/components/form/DynamicForm.tsx [hunk 2]
```diff
@@ -12,33 +13,27 @@ const { width } = Dimensions.get('window');
...
 	formStyle: FormStyles;
 	formSchema: any;
 	presetValues?: any[];
 	showActionSheetWithOptions?: any;
+	screenProps: any;
 }
 
 export interface State {
-	formData: any;
+	// formData: any;
```
Comment (Nicolaas0411) [inline src/components/form/DynamicForm.tsx]: remove comment

File: src/components/form/DynamicForm.tsx (hunk 1 above)
Comment (mullerfourie) [reply src/components/form/DynamicForm.tsx]: Will do, thanks Nick.

File: config.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 export const env = {
+    REACT_IXO_LOCAL_ENV_IP: "192.168.1.253",
```
Comment (brennonh) [inline config.tsx]: replace hard coded ip addresses with DNS

**Review by brennonh — APPROVED** [review]

**Pull request closed without merging**